x-pack/test/functional/apps/aiops/explain_log_rate_spikes.ts

```diff
@@ -79,6 +146,7 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
       it(`${farequoteDataViewTestData.suiteTitle} loads the explain log rate spikes page`, async () => {
         // Start navigation from the base of the ML app.
         await ml.navigation.navigateToMl();
+        await elasticChart.setNewChartUiDebugFlag(true);
```

Comment:
In other test where we use this, we disable the debug flag again `await elasticChart.setNewChartUiDebugFlag(false);` in the `after` method. I think we might want to do the same here.

Reply:
Added in [d5b1407](https://github.com/elastic/kibana/pull/138661/commits/d5b14078cbcc68c9411c55f6ec94fe9f304788f0).